packages/webapp/src/middleware/alchemyWebsocket.js

```diff
@@ -43,24 +46,33 @@ const alchemyWebsocketMiddleware = () => {
 
   const newBlockSubscriptionRequest = JSON.stringify({
     "jsonrpc":"2.0",
-    "id": blockId,
+    "id": '44',
     "method": "eth_subscribe",
     "params": ["newHeads"]
   });
 
+  const newTxSubscriptionRequest = JSON.stringify({
+    "jsonrpc": "2.0",
+    "id": '45',
+    "method": "eth_subscribe",
+    "params": [
+      "alchemy_pendingTransactions",
+      { toAddress: [auctionAddress] },
+    ]
+  });
 
   // Define the Handler Methods
   const handleOpen = store => () => {
     console.log('Alchemy Web Socket OPEN.');
-    store.dispatch(setEthereumConnected(true));
     socket.send(newBlockSubscriptionRequest);
+    store.dispatch(setEthereumConnected(true));
+    socket.send(newTxSubscriptionRequest);
+    store.dispatch(setMempoolListening(true));
```

Comment:
Is this still necessary? Do we need to know in state that mempool is listening if we're handling every event?

Reply:
Thanks! Removed this one!